### DISCUSSION TITLE
larger guns. and increased mass limit on rotors.

### DISCUSSION THREAD

**Discussion by eageus** [tsunamayo/Starship-EVO#4615] [Suggestions]
i have a 46km ship. its guns ant even able to shoot half of its own length. i speak not just for myself wen i say i think we need 32m or at least 16m weapons. and much larger mass limits on rotors. in my opinion 32m rotors should be able to hold and spin a turret of 512mx512m. or add 3x3 block rotors that way we can have these super large turrets or other things on rotors. i would also love to have 32m system blocks. thrusters reactors etc. so we can build super large ships. without having to use 4m system blocks. would also like increased weapon range for all scales. im not the only one that was wanted some of these changes. ive talked to a few people that feel the same way.

**Comment by tsunamayo** [comment]
@eageus Hi, could you send me the blueprint so I can have a look at it? 46km seems like a lot...

**Reply by eageus** [reply]
yea sure no problem. here it is
[new mega titan.zip](https://github.com/tsunamayo/Starship-EVO/files/8199959/new.mega.titan.zip)

**Comment by eageus** [comment]
@tsunamayo don't want to bother you i know you are super busy but i really want some sort of answer to my suggestion. i really really want to be able to use my 46km ship in some way. along with its larger turret i am working on.

**Reply by eageus** [reply]
@tsunamayo ??

**Reply by Arutus** [reply]
dont expect a reply anytime soon mans ultra busy

**Comment by ghost** [comment]
> i have a 46km ship. its guns ant even able to shoot half of its own length. i speak not just for myself wen i say i think we need 32m or at least 16m weapons. and much larger mass limits on rotors. in my opinion 32m rotors should be able to hold and spin a turret of 512mx512m. or add 3x3 block rotors that way we can have these super large turrets or other things on rotors. i would also love to have 32m system blocks. thrusters reactors etc. so we can build super large ships. without having to use 4m system blocks. would also like increased weapon range for all scales. im not the only one that was wanted some of these changes. ive talked to a few people that feel the same way.

I also love building massive structures. I had the same issue with weapons 🫤 was wondering why they aren’t scalable to be even larger.  Same with thrust to be honest. Building a capital ship ain’t easy 😳.

also, since you brought up things like hinges. Why can’t they go into the negative? Having to ensure a hinge is on the “correct” way only to realize the game won’t let it go “negative degrees” through logic simply blew my mind :(
